authors/apps/articles/urls.py

```diff
@@ -1,14 +0,0 @@
-from django.urls import path
```

Comment:
Why are you deleting this code?

Reply:
I rebased from Develop and I had to sort out a merge conflict with Gidrafs work that was merged earlier for rate articles. The issue was I had a folder called article and he had articles. Soo we had to combine both to one folder. I therefore took his code and integrated it with mine in the article folder then got rid of his folder. That is why there is a-lot of deleted work. Both his work and mine are working perfectly after the conflict.